web/src/demo_organizations_ui.ts

```diff
@@ -26,13 +28,48 @@ export function get_demo_organization_deadline_days_remaining(): number {
 }
 
 export function insert_demo_organization_warning(): void {
-    const days_remaining = get_demo_organization_deadline_days_remaining();
-    const rendered_demo_organization_warning = render_demo_organization_warning({
-        is_demo_organization: realm.demo_organization_scheduled_deletion_date,
-        is_owner: current_user.is_owner,
-        days_remaining,
+    const demo_organization_warning_container = render_demo_organization_warning_container({
+        is_demo_organization: true,
     });
-    $(".organization-box").find(".settings-section").prepend($(rendered_demo_organization_warning));
+    $(".organization-box")
+        .find(".settings-section")
+        .prepend($(demo_organization_warning_container));
+    const days_remaining = get_demo_organization_deadline_days_remaining();
+    let buttons: ActionButton[] = [
+        {
+            attention: "borderless",
+            label: $t({defaultMessage: "Learn more"}),
+            custom_classes: "demo-organizations-help",
+        },
+    ];
+    if (current_user.is_owner) {
+        buttons = [
+            ...buttons,
+            {
+                attention: "quiet",
+                label: $t({defaultMessage: "Convert"}),
+                custom_classes: "convert-demo-organization",
+            },
+        ];
+    }
+    const demo_organization_warning_banner: banners.Banner = {
+        intent: days_remaining <= 7 ? "danger" : "info",
+        label: new Handlebars.SafeString(
+            $t_html(
+                {
+                    defaultMessage:
+                        "This demo organization will be automatically deleted in {days_remaining} days, unless it's converted into a permanent organization.",
+                },
+                {
+                    days_remaining,
+                },
+            ),
+        ),
```

Comment:
Nothing critical, but we don't need `$t_html()` over here and can just do away with `$t()` translation tag since no html is involved here anymore.

Reply:
Thanks for catching that! Added a prep commit to clean up the nabar alert for demo organizations too.